### PR TITLE
Usage and man page update for rspconfig dump

### PR DESCRIPTION
**Pull request by gurevichmark**
#4381 

Add usage and man pages for `rspconfig dump` command

### CODE REVIEW

File: xCAT-client/pods/man1/rspconfig.1.pod [hunk 1]
```diff
@@ -813,6 +820,30 @@ Output is similar to:
...
+ p9euh02: [1] Generated: 09/06/2017 14:31:49, Size: 4528
+ p9euh02: [2] Generated: 09/06/2017 14:31:55, Size: 4516
+ p9euh02: [3] Generated: 09/06/2017 14:32:01, Size: 4236
+ p9euh02: [4] Generated: 09/06/2017 14:32:07, Size: 4248
+ p9euh02: [5] Generated: 09/06/2017 14:32:11, Size: 4268
+
+=item 32.
+To generate and download BMC dump:
+
+ rspconfig p9euh02 dump -l
```
Comment (whowutwut) [inline xCAT-client/pods/man1/rspconfig.1.pod]: I think this one has an extra `-l` option?

File: xCAT-client/pods/man1/rspconfig.1.pod [hunk 2]
```diff
@@ -324,6 +326,11 @@ Select whether each memory bank should be enabled or disabled. State changes tak
 
 The subnet mask.
 
+=item B<dump>
+
+Manage OpenBMC system dumps. If no suboption is provided, will generate, wait and download the dump files to /var/log/xcat/dump.
```
Comment (whowutwut) [inline xCAT-client/pods/man1/rspconfig.1.pod]: Suggest: 



`If no sub-option is provided, the command will generate, wait, and download the dump.`

we specify the management and service nodes later on in this, so maybe just put it once.

File: xCAT-client/pods/man1/rspconfig.1.pod [hunk 3]
```diff
@@ -324,6 +326,11 @@ Select whether each memory bank should be enabled or disabled. State changes tak
...
+B<-c> will clear a single or all dumps on the BMC. B<-l> will list all the generated dumps on the BMC. B<-g> will generate a new dump on the BMC. Dump generation can take a few minutes. B<-d> will download a single dump from the BMC to /var/log/xcat/dump on management or service node.
```
Comment (whowutwut) [inline xCAT-client/pods/man1/rspconfig.1.pod]: Should these be in the same line or separate lines?

Comment (whowutwut) [reply xCAT-client/pods/man1/rspconfig.1.pod]: Suggest: 

`will clear a single specified dump, or use 'all' to clear all dumps on the BMC.`

**Comment by gurevichmark** [comment]
Suggested changes made under a new commit

**Pull request merged**